toolchain/sem_ir/inst_namer.cpp

```diff
@@ -644,7 +644,7 @@ auto InstNamer::NamingContext::AddInstName(std::string name) -> void {
       loc_id_or_fingerprint = LocId(inst_id_);
     }
     auto scoped_name = inst_namer_->GetScopeInfo(scope_id_).insts.AllocateName(
-        *inst_namer_, loc_id_or_fingerprint, name);
+        *inst_namer_, loc_id_or_fingerprint, std::move(name));
```

Comment:
This doesn't move the runtime of this test, but it is a good idea, and is done in other callers of AllocateName.